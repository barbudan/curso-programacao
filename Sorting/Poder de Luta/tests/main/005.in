10
gokuz 100
goku 100
gokua 100
gok 100
go 100
g 100
blablabla 120
bli 190
dan 9000
hello 9000