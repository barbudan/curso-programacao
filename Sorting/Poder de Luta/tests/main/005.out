dan 9000
hello 9000
bli 190
blablabla 120
g 100
go 100
gok 100
goku 100
gokua 100
gokuz 100